README.md

```diff
@@ -42,9 +42,11 @@ As a starting point for your own project take a look at https://github.com/ether
 
 ##### Running from command line:
 ```
-> git clone https://github.com/ethereum/ethereumj
-> cd ethereumj
-> ./gradlew run [-PmainClass=<sample class>]
```

Comment:
But then I’d put it into another section as otherwise it would confuse sb. who just wants to build a JAR.